src/plugins/CouchDBSearchFolder/plugin.js

```diff
@@ -1,21 +1,26 @@
 export default function (folderName, couchPlugin, searchFilter) {
+  const DEFAULT_NAME = 'CouchDB Documents';
+
   return function install(openmct) {
     const couchProvider = couchPlugin.couchProvider;
+    //replace any non-letter/non-number with a hyphen
+    const couchSearchId = (folderName || DEFAULT_NAME).replace(/[^a-zA-Z0-9]/g, '-');
```

Comment:
Would there ever be a situation where folderName isn't passed in? seems unlikely if couchPlugin and searchFilter are expected?

Reply:
Correct. But this was in case someone put in empty string or something.